com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineProjectorEditor.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    [CustomEditorForRenderPipeline(typeof(Projector), typeof(UniversalRenderPipelineAsset))]
+    [CanEditMultipleObjects]
+    public class UniversalRenderPipelineProjectorEditor : Editor
```

Comment:
This class doesn't need to be public. Otherwise we have to document / generate API docs for it.